src/python/grpcio_tests/tests/unit/credentials/cert_hier_1/certs/ca.cert.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Avoid abbreviations, acronyms, and initialisms. Can the directories for these files be just "certificate_hierarchy_1" and so forth?

Reply:
of course! will do

Reply:
done. there's the abbreviations in the code, e.g., `def cert_hier_2_root_ca_cert()`; unclear if you want to change those as well?

Reply:
Yes please.